nnpdf_data/nnpdf_data/commondata/CMS_WPWM_8TEV_MUON/uncertainties.yaml

```diff
@@ -218,8 +218,8 @@ bins:
   corr_stat_10: 4.59724656e+02
   corr_stat_11: -1.19020891e+03
   corr_stat_12: -7.78086020e+02
-  corr_stat_13: -3.25115482e+02
-  corr_stat_14: -1.74120972e+01
+  corr_stat_13: 3.25115482e+02
+  corr_stat_14: 1.74120972e+01
   corr_stat_15: 1.85441751e+02
```

Comment:
One thing to note here is that after re-running the filter without any modification, the signs of these two sources flip. I don't know the reason, but as long as the same sing-flip is propagated for all bins and consistently always with the same sources, then it should not be a problem.